src/main/java/Step3/RacingGame.java

```diff
@@ -7,52 +7,73 @@
 public class RacingGame {
 
     private final List<RacingCar> cars;
-    private int playNum;
-    private boolean hasBoard;
+    private final List<List<Integer>> gameRecord;
+    private final int playNum;
 
-    public RacingGame() {
+    public RacingGame(int carNum, int playNum) {
         this.cars = new ArrayList<>();
-        this.playNum = 0;
-        this.hasBoard = false;
-    }
-
-    public void makeBoard(int carNum, int playNum) {
-        this.playNum = playNum;
-
-        this.cars.clear();
+        this.gameRecord = new ArrayList<>();
 
         for (int i = 0; i < carNum; i++) {
             this.cars.add(new RacingCar());
         }
 
-        this.hasBoard = true;
+        this.playNum = playNum;
     }
 
     public List<RacingCar> getCars() {
         return this.cars;
     }
 
-    public boolean play() {
-        if (!this.hasBoard) {
-            return false;
+    public List<List<Integer>> play() {
+        Random random = new Random();
+        for (int i = 0; i < this.playNum; i++) {
+            List<Integer> turnRecord = tryMoveCars(random);
+
+            this.gameRecord.add(turnRecord);
         }
 
-        System.out.println("실행 결과");
+        return this.gameRecord;
+    }
+
+    public List<Integer> getWinner() {
+        List<Integer> winnersIdx = new ArrayList<>();
+        List<Integer> lastTurnRecord = this.gameRecord.get(this.gameRecord.size() - 1);
 
-        Random random = new Random();
-        for (int i = 0; i < this.playNum; i++) {
-            moveCar(random);
-            System.out.println();
+        int max = 0;
+        for (Integer finalPosition : lastTurnRecord) {
+            max = getMax(finalPosition, max);
         }
 
-        return true;
+        for (int i = 0; i < lastTurnRecord.size(); i++) {
+            pickIfWinner(lastTurnRecord, i, max, winnersIdx);
+        }
+
+        return winnersIdx;
+    }
+
+    private static void pickIfWinner(List<Integer> lastTurnRecord, int i, int max, List<Integer> winnersIdx) {
```

Comment:
```suggestion
    private void pickIfWinner(List<Integer> lastTurnRecord, int i, int max, List<Integer> winnersIdx) {
```
static은 제거되어도 될 것 같아요 :) `getMax`도입니다.